src/statistics.jl

```diff
@@ -327,7 +327,7 @@ function apply_statistic(stat::HistogramStatistic,
             x_min = Gadfly.concrete_minimum(values)
             span = x_max - x_min
             binwidth = span / d
-            bincounts ./= sum(bincounts) * binwidth
+            bincounts = bincounts ./ sum(bincounts) * binwidth
```

Comment:
It's not. This line causes an `InexactError` in the case that `binwidth` is a float. `bincounts` is always an integer array and with the new meaning of `./=` on v0.5 we are trying to do an in-place update of integer array using floats causing a type coercion error, i.e. the `InexactError`.

julia v0.4: 
```julia
julia> a = [1,2,3]
3-element Array{Int64,1}:
 1
 2
 3

julia> a ./= 2.0
3-element Array{Float64,1}:
 0.5
 1.0
 1.5
```

julia v0.5-rc3
```julia
julia> a = [1,2,3]
3-element Array{Int64,1}:
 1
 2
 3

julia> a ./= 2.0
ERROR: InexactError()
 in copy!(::Base.LinearFast, ::Array{Int64,1}, ::Base.LinearFast, ::Array{Float64,1}) at ./abstractarray.jl:559
 in broadcast!(::Base.#identity, ::Array{Int64,1}, ::Array{Float64,1}) at ./broadcast.jl:24
```